rpcserver.go

```diff
@@ -4122,6 +4123,15 @@ func createRPCOpenChannel(r *rpcServer, dbChannel *channeldb.OpenChannel,
 	// Fetch the set of aliases for the channel.
 	channelAliases := r.server.aliasMgr.GetAliases(dbScid)
 
+	peerAlias := ""
+	if peerAliasLookup {
+		alias, err := r.server.graphDB.LookupAlias(nodePub)
+		if err != nil {
+			alias = ""
```

Comment:
I guess in case of an error here we should not abort the flow but continue with an empty alias? But maybe we could add a debug message here?

Reply:
I think the behavior (and name of the flag and so on) should be exactly the same as in `fwdinghistory`. See https://github.com/lightningnetwork/lnd/pull/7001#discussion_r994223661.

Reply:
Thanks, yes. I mirrored the error behavior as in the PR you mentioned. The flag name was later changed in another commit however, here: https://github.com/lightningnetwork/lnd/pull/7083.